modules/Oculus_module/src/TorsoRetargeting.cpp

```diff
@@ -0,0 +1,85 @@
+/**
+ * @file HeadRetargeting.cpp
+ * @authors Giulio Romualdi <giulio.romualdi@iit.it>
+ *          Mohamed Babiker Mohamed Elobaid <mohamed.elobaid@iit.it>
+ * @copyright 2018 iCub Facility - Istituto Italiano di Tecnologia
+ *            Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+ * @date 2018
+ */
+
+// iDynTree
+#include <iDynTree/Core/EigenHelpers.h>
+#include <iDynTree/yarp/YARPConversions.h>
+#include <iDynTree/yarp/YARPEigenConversions.h>
+
+#include <TorsoRetargeting.hpp>
+#include <Utils.hpp>
+
+
+// This code was taken from https://www.geometrictools.com/Documentation/EulerAngles.pdf
```

Comment:
If the torso `inverse kinematics` is not used you can remove this line. 
Furthermore, I suggest using `iDynTree` library for evaluating forward kinematics (and inverse kinematics) if needed. 
As regards forward kinematics you can use [`iDynTree::KinDynComputations`](http://wiki.icub.org/codyco/dox/html/idyntree/html/classiDynTree_1_1KinDynComputations.html) while for inverse kinematics there is the new library developed by @lrapetti in `iDynTree`. 
I know that I'm annoying but using a library that depends only on the `urdf` model allows to have a code more scalable with different robots. @s-dafarra

Reply:
I opened an issue #20  to use [iDynTree::KinDynComputations](http://wiki.icub.org/codyco/dox/html/idyntree/html/classiDynTree_1_1KinDynComputations.html) Oculus Module instead of doing it manually.

Reply:
@GiulioRomualdi  if you agree I will add it as an improvement soon. I think this issue does not need to block this PR. What do you think?

Reply:
I am working on this in another issue, as I mentioned before. When it is done, I will open another PR for that. So, if you agree, I resolve this conversation.